### ISSUE TITLE
Cannot create new Server

### ISSUE THREAD

**Issue by kckanav** [phpmyadmin/phpmyadmin#16537]
### Describe the bug

The bug is not allowing me to create a new server. It is throwing a fatal error when I am trying to create it for my website.

### To Reproduce

1. Go to /phpmyadmin/setup (phpmyadmin is the shortcut for phpMyAdmin-5.0.4)
2. Click on New Server. I definedmy hostname, port, yes to use SSL. Then, on authentication, I select config, and put in user and password for authentication. When I click apply, I am getting this error.

### Expected behavior

I expected it to add the server with an error which I was told to ignore by the server owner. 

### Screenshots

As soon as I click apply, this shows up as plain text on the browser. 
```
Fatal error: Uncaught TypeError: mysqli_connect() expects parameter 4 to be string, null given in phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php:226
Stack trace: #0 phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php(226): mysqli_connect('xxxxx', 'root', 'xxxxx', NULL, 1707, NULL)
#1 phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php(304): PhpMyAdmin\Config\Validator::testDBConnection('xxxxx', 1707, NULL, 'root', 'xxxx', 'Server')
#2 [internal function]: PhpMyAdmin\Config\Validator::validateServer('Server', Array)
#3 phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php(142): call_user_func_array('PhpMyAdmin\\Conf...', Array)
#4 phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/FormDisplay.php(201): PhpMyAdmin\Config\Validator::validate in phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php on line 226
```

### Server configuration

- Operating system:  UNIX/LINUX
- phpMyAdmin version: 5.0.4

**Comment by ibennetch** [comment]
I believe this has already been fixed to be part of the 5.1.0 release. I just released 5.1.0-rc1 (a release candidate version); could you try with that version to confirm whether the fix works in your situation?

**Comment by williamdes** [comment]
I searched and this is now fixed in 5.1-rc1 by f2e49a92c6681414b74140d98ea92607db791c22 (https://github.com/phpmyadmin/phpmyadmin/commit/f2e49a92c6681414b74140d98ea92607db791c22#diff-08d19463f2535e34c30108f532ad5190b89b5aefab8ce407c8905d2b4820e206)
